modules/sb-start-asb.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * applications/service_brokers/installing-ansible-service-broker.adoc
+
+[id='sb-start-asb-{context}']
+= Starting the {asb-name}
+
+Once you have installed the {asb-name} Operator, you can start the {asb-name} using the following procedure.
+
+.Prerequisites
+
+* You have enabled service catalog.
+* You have installed the {asb-name} Operator.
+
```

Comment:
'Automation Broker Operator'

Reply:
As shawn-hurley said in other place, please ignore this comments at present. We will double confirm while downstream ready.